amaranth_soc/csr/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .bus import *
 from .event import *
+from .reg import *
```

Comment:
It would be good to have also `from . import field` so e.g. `csr.field.RW` can be accessed easily and like the illustrated examples in the RFC. It's worth nothing that (in a user program) `from amaranth_soc import csr.field` is a syntax error.

I think `from .field import *` would pollute the namespace too much as the names are rather generic, but that is an alternative too.

Reply:
This is fixed now.